modules/install/pages/extensions.adoc

```diff
@@ -1,24 +1,316 @@
+-rw-r--r--. 1 jboss jboss  14802 Apr  9 04:14 application_10.xsd
+-rw-r--r--. 1 jboss jboss   9058 Apr  9 04:14 application_1_4.xsd
+-rw-r--r--. 1 jboss jboss   9803 Apr  9 04:14 application_5.xsd
+----
+
+* Subsystems can be configured using Management Console or CLI, so there is no need to edit the XML directly to configure a subsystem. Below screenshot shows the subsystem configuration in Management Console.
++
+image::sub.png[align="center"]
+
+* If a subsystem does not require any specific settings, an empty <subsystem> entry is still required in the configuration file. For example, the jaxrs subsystem is configured by default, without any specific settings:
++
+[subs="+quotes,+macros"]
+----
+<subsystem xmlns="urn:jboss:domain:jaxrs:1.0"/>
+----
+
+* The JBoss EAP standalone server instance configuration file `standalone.xml` contains a single, anonymous, profile definition.
+
+* The `domain.xml` managed domain configuration file contains four pre-defined profiles by defauilt:
+
+. **default**: Is the most commonly used subsystems, including logging, security, datasources, infinispan, weld, webservices, and ejb3. The default implements not only the Jakarta EE Web Profile, but also most of the Jakarta EE Full Profile.
+
+. **ha**: Contains the exact same subsystems as the default profile, with the addition of clustering capabilities, provided mainly by the jgroups subsystem.
+
+. **full**: Is similar to the default profile, but notably adds the messaging (messaging-activemq) and a few other less used subsystems.
+
+. **full-ha**: Is the same as the full profile, but with the addition of clustering capabilities.
+
+* Look in the `JBOSS_HOME/standalone/configuration` directory. Notice there are four standalone configuration files:
+
+. **standalone.xml**: Compares to the default profile in domain.xml.
+
+. **standalone-ha.xml**: Compares to the ha profile in domain.xml.
+
+. **standalone-full.xml**: Compares to the full profile in domain.xml.
+
+. **standalone-full-ha.xml**: Compares to the full-ha profile in domain.xml.
+
+They are provided so a standalone server instance can easily be started with more or less subsystems available.
+
+[NOTE]
+====
+Be careful not to confuse a JBoss EAP profile with a Jakarta EE profile. A JBoss EAP profile is a collection of subsystem configurations that are used to define the capabilities and services of a JBoss EAP server instance. A Jakarta EE profile is a collection of Java EE standards.
+====
+
+* In a managed domain, the administrator can create new profiles, either from scratch, or cloned from the ones provided, and then customize the new profiles before associating them to their respective groups.
+
+* A profile can also be a child of another profile, thus inheriting the subsystem configuration from its parent profile, so common configurations can be shared and maintained in a single place.
+
+== Lab 1: Starting different Subsystems
```

Comment:
redudant and repeating. Remove this lab. Just include the steps to start one of the profiles as part of the lecture itself.

Reply:
removed